questions/urgent_items_vs_process_improvement.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "Urgent items vs process improvement"
+parent: Katas
+---
+The Product Owner asks the Developers to pick up a very urgent item late in Sprint that was not in the forecast, nor is it related to the Sprint Goal. The Developers believe it can pick this up, as it is close to meeting the Sprint Goal. but this would involve not meeting their process improvement goal agreed upon during the last Sprint Retrospective. The Product Owner argues that, as it’s the highest priority to satisfy the customer, the needs of the customer have a higher priority than the process improvement goal for the team.
```

Comment:
```suggestion
The Product Owner asks the Developers to pick up a very urgent item late in Sprint that was not in the forecast, nor is it related to the Sprint Goal. The Developers believe it can pick this up, as it is close to meeting the Sprint Goal. But this would involve not meeting their process improvement goal agreed upon during the last Sprint Retrospective. The Product Owner argues that, as it’s the highest priority to satisfy the customer, the needs of the customer have a higher priority than the process improvement goal for the team.
```